packages/fast-element/src/observation/subscriber-collection.ts

```diff
@@ -0,0 +1,94 @@
+export interface ISubscriber {
+    handleChange(source: any, args: any): void;
+}
+
+export class SubscriberCollection {
+    private sub1: ISubscriber | undefined = void 0;
+    private sub2: ISubscriber | undefined = void 0;
+    private sub3: ISubscriber | undefined = void 0;
+    private spillover: ISubscriber[] | undefined = void 0;
```

Comment:
might be nice to just stick a comment in here as to why we're doing the 3 subscriber then spillover as you have in the PR description

Reply:
Added a summary and remarks to the SubscriberCollection that captures this info.